lua-plugins/cli_filter.lua.in

```diff
@@ -199,8 +199,8 @@ function slurm_cli_pre_submit(options, offset)
         return partition == 'gpu' or partition == 'gpu-dev' or partition == 'gpu-highmem' or partition == 'mwa-gpu'
     end
 
-    local function is_acceptance_partition(partition)
-        return partition == 'acceptance'
+    local function is_excepted_partition(partition)
```

Comment:
I'm unsure what the quantum partition has to do with the acceptance partition, nor do I know why the function name is being changed?
The acceptance partition was setup to allow jobs to be launched on any node and use features instead of partition names as part of acceptance and return to service testing practices.

I think to prevent potential future confusion the quantum partition should have it's own function to encapsulate it and any future quantum sub-partitions and keep it separate from the acceptance partition (which should include all partitions on Setonix).

Reply:
Thanks for the clarification.
From looking at the CR it seems this was done as a hack rather than adding the quantum partition to the acceptance.
I'll check with David to see if there's any reason this shouldn't be done instead of making the hack a permanent fixture.

Reply:
Righto!

Reply:
Sorry, I just realised that adding this to the acceptance partition will make no difference.

This change looks fine.